projects/libs/core-library-typescript/src/common-library/logger.ts

```diff
@@ -15,7 +15,7 @@ export interface LogzIoOptions {
 export interface FileOptions {
   fileName: string;
   json?: boolean;
-  maxSize?: number;
+  maxsize?: number;
```

Comment:
This is our object, which we later translate to Winston's weird key names, so it should be according to our naming conventions (`maxSize` and not `maxsize`).

Reply:
Fixed.